Task was destroyed but it is pending!
task: <Task pending coro=<member_reminder() running at /home/discord/discord/website/paypal-discord-bot-KickConnectz.py:244> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fb08d7c6eb8>()]>>
Task was destroyed but it is pending!
task: <Task pending coro=<member_invite() running at /home/discord/discord/website/paypal-discord-bot-KickConnectz.py:177> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fb08d7c6c48>()]>>
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 16:36:36.690785) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 17:27:00.860340) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 19:17:15.522835) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 20:08:47.414530) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 21:55:36.445028) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-20 22:19:06.842488) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-21 03:36:07.872872) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-21 05:51:32.817418) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-21 05:58:55.420067) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-21 11:33:35.502257) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-21 12:06:04.694509) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-KickConnectz.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-KickConnectz.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Task exception was never retrieved
future: <Task finished coro=<member_reminder() done, defined at /home/discord/discord/website/paypal-discord-bot-KickConnectz.py:180> exception=InvalidArgument('Destination must be Channel, PrivateChannel, User, or Object. Received NoneType',)>
Traceback (most recent call last):
  File "/home/discord/discord/website/paypal-discord-bot-KickConnectz.py", line 205, in member_reminder
    await client.send_message(user, embed=embed_message("Reminder", str(bot_message.first_reminder).format(member.discord_username)))
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 1145, in send_message
    channel_id, guild_id = yield from self._resolve_destination(destination)
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 289, in _resolve_destination
    raise InvalidArgument(fmt.format(destination))
discord.errors.InvalidArgument: Destination must be Channel, PrivateChannel, User, or Object. Received NoneType
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-22 00:13:35.435335) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-22 04:25:00.204554) while time zone support is active.
  RuntimeWarning)
/home/discord/venv/lib/python3.6/site-packages/django/db/models/fields/__init__.py:1421: RuntimeWarning: DateTimeField Member.subscription_date_expire received a naive datetime (2019-06-22 10:26:57.256147) while time zone support is active.
  RuntimeWarning)
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 307, in _run_event
    yield from getattr(self, event)(*args, **kwargs)
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 279, in on_message
    activate_status = activate_user(message.author, email.group(0))
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 92, in activate_user
    if member.subscription_date_expire:
AttributeError: 'NoneType' object has no attribute 'subscription_date_expire'
Task exception was never retrieved
future: <Task finished coro=<member_reminder() done, defined at /home/discord/discord/website/paypal-discord-bot-hexane.py:180> exception=InvalidArgument('Destination must be Channel, PrivateChannel, User, or Object. Received NoneType',)>
Traceback (most recent call last):
  File "/home/discord/discord/website/paypal-discord-bot-hexane.py", line 205, in member_reminder
    await client.send_message(user, embed=embed_message("Reminder", str(bot_message.first_reminder).format(member.discord_username)))
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 1145, in send_message
    channel_id, guild_id = yield from self._resolve_destination(destination)
  File "/home/discord/venv/lib/python3.6/site-packages/discord/client.py", line 289, in _resolve_destination
    raise InvalidArgument(fmt.format(destination))
discord.errors.InvalidArgument: Destination must be Channel, PrivateChannel, User, or Object. Received NoneType
